sphinx/writers/latex.py

```diff
@@ -368,6 +369,17 @@ def __init__(self, document, builder):
             'releasename':  _('Release'),
             'indexname':    _('Index'),
         })
+        if builder.config.latex_engine == 'lualatex':
+            self.elements['luatex'] = \
```

Comment:
I think the value of `luatex` will not be customized by both users and sphinx.
So I would like to modify this like following
```
<% if latex_engine == 'luatex' -%>
\IfFileExists{luatex85.sty}{...}
<% endif -%>
```
(at this moment, `latex_engine` is not given to templates. So we should add it as a template variable)

What do you think?